src/components/elements/MenuLink.js

```diff
@@ -0,0 +1,35 @@
+import React from 'react'
+import { Link } from 'react-router-dom'
+import MenuItem from 'material-ui/MenuItem'
+import ActionDelete from 'material-ui/svg-icons/action/delete'
+
+export const MenuLink = (type, id, onClick, dispatch) => {
+  validateType(type)
```

Comment:
Couldn't this be done with props? I think there's a way to write custom props to handle this sort of thing, and that will work a little nicer in the error stack.

Reply:
The way the title portion is written requires that there be at least 2 characters.